chsdi/views/historicalmaps.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.view import view_config
+
+from pyramid.renderers import render_to_response
+import pyramid.httpexceptions as exc
+
+
+@view_config(route_name='historicalmaps')
+def luftbilder(request):
```

Comment:
You should do as much as you can here, move all you can from the makos here instead.